lib/main.js

```diff
@@ -1,85 +1,64 @@
 'use strict';
 
-var _		= require('lodash');
-var request	= require('request');
-
-var dashboard = require('./oo_dashboard');
-var executions = require('./oo_executions');
-var schedules = require('./oo_schedules');
-var config = require('./oo_config');
-
-/**
- * OO Object declaration
- *
- */
-var OO = {};
-
-/** 
- * OO Object
- *
- * @param {object} options
- * 		required:
- *	 		username - username for OO that is allowed to use the API
- *			password - the password for the username
- *			validSSL - true or false whether to require a valid SSL certificate or allow self-signed certs
- *			baseUrl  - the base url of the REST endpoint for an OO server, for example: http://localhost:8050/oo/rest/v1
- */
-OO.setClient = function(options) {
-
-	if (!options.username || !options.password) {
-		return new Error("no username or password settings provided in options object");
+import 'babel-polyfill';
```

Comment:
I don't mind that much the import, just don't think it's a good idea to adapt it yet when it hasn't got any real support from Node.js, and currently is just syntactic sugar. Although, the same can probably be said about class notation so I'm not religious about import/export :-)

Maybe I wasn't clear before so let me try that again:
1. We can keep the library code itself as being the most bleeding edge possible ES7 and other exotic charms, which we don't need babel as part of the code-base. This repository isn't expected to be cloned and used as standalone so anyone who would actually work on it are devs, and they *can* be expected to develop the library with any standards we define (i.e: ES*)
2. The output of the build for the library, that should indeed transpile, polyfill and whatnot in order to make the library's output compatible with older Node.js versions.

That's why I was referring to the babel-polyfill inside the actual library code (not the dist/index.js output). Actually, I think you need that for import/export, no?